### PR TITLE
Improve logging in blunderbuss

### PR DESCRIPTION
**Pull request by jmguzik**
Add debug log and improve error log to trace problem with multiple graphql queries

/cc @petr-muller 

Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

File: prow/plugins/blunderbuss/blunderbuss.go [hunk 1]
```diff
@@ -336,12 +336,13 @@ func findReviewer(ghc githubClient, log *logrus.Entry, useStatusAvailability boo
 		}
 		busy, err := isUserBusy(ghc, candidate)
 		if err != nil {
-			log.Errorf("error checking user availability: %v", err)
+			log.Errorf("error checking user %s availability: %v", candidate, err)
```
Comment (petr-muller) [inline prow/plugins/blunderbuss/blunderbuss.go:339]: Let's log this in a structured way (also a good opportunity to log the *error* in a structured way): 

`log.WithField(...).WithError(err).Error("message")`

Same for the second message.

**Review by petr-muller — APPROVED** [review]

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**